recipes/leg2cheb/meta.yaml

```
@@ -0,0 +1,62 @@
+{% set build = 0 %}
+{% set name = "leg2cheb" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/spectralDNS/Legendre-to-Chebyshev/archive/{{ version }}.tar.gz
+  sha256: 3ff70cecdb98cb851dc8b04709c616cdd6b3b681dee83c0c8b022a6422718189
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  script: 
+    - cd src
+    - meson setup build ${MESON_ARGS}
+    - meson compile -C build
+    - meson install -C build
+
+requirements:
+  build:
+    - numpy >=2.0                              
+    - setuptools                          
+    - python                              
+    - cython                              
+    - pip                                 
+    - meson                               
+    - ninja                               
+    - pkg-config         
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]            
+    - compilers
```

compilers is a package only for endusers, please use the compiler functions in recipes.
```suggestion
    - {{ compiler('c') }}
```